cloud/pkg/dynamiccontroller/application/application.go

```diff
@@ -369,7 +369,6 @@ func msgToApplication(msg model.Message) (*Application, error) {
 	return app, nil
 }
 
-// TODO: upgrade to parallel process
```

Comment:
is this pr support parallelly process application?

Reply:
Parallel processing was not resolved, it may have been accidentally deleted.

But I found areas in the `Process` where there might be concurrency issues.

```go
err := c.HandlerCenter.AddListener(app.ToListener(*option))
```
concurrent map read and map write
https://github.com/kubeedge/kubeedge/blob/1678686e9b557ccb0c2c57985954604c13323372/cloud/pkg/dynamiccontroller/application/eventhandler.go#L140-L163

Is this the problem to be solved if parallel processing is to be implemented, or are there some other problems

Reply:
@Iceber Yes, I think `concurrent map read and map write` is one of the problems in parallelly processing application. This only influence **Processing Watch**, and has no relationship to Get and the Write(create delete update  patch). We need think more and could fix it in next pr. 
:)